discovery-provider/src/tasks/users.py

```diff
@@ -211,8 +213,37 @@ def get_metadata_overrides_from_ipfs(session, update_task, user_record):
         if ipld_blacklist_entry:
             return None
 
+        # Manually peer with user creator nodes
+        update_ipfs_peers_from_user_endpoint(
```

Comment:
@hareeshnagaraj This is basically logic to get the creator node endpoint from the current user and peer with it before attempting to retrieve metadata in case it's a new user not in the db?

Reply:
thats right